packages/backend/src/api/request.ts

```diff
@@ -30,10 +30,12 @@ export type ClerkBackendApiResponse<T> =
   | {
       data: T;
       errors: null;
+      totalCount?: number;
     }
   | {
       data: null;
```

Comment:
@panteliselef `data` can be `null` because the response body data can be `null`.
We probably need to return an empty array when we have an array type? Would this help the hooks code using the APIs get cleaner?

reply to: https://github.com/clerk/javascript/pull/2199/files#r1404220574